src/mesh/difops.cxx

```diff
@@ -784,7 +789,7 @@ const Field2D bracket(const Field2D &f, const Field2D &g, BRACKET_METHOD method,
     result = 0.0;
   }else {
     // Use full expression with all terms
-    result = b0xGrad_dot_Grad(f, g) / f.getCoordinates(result_loc)->Bxy;
+    result = b0xGrad_dot_Grad(f, g, outloc) / f.getCoordinates(result_loc)->Bxy;
```

Comment:
presumably outloc and result_loc must be the same here?

Reply:
Currently that's what `bracket_location` does. I think it's another leftover from trying to interpolate the result of derivatives. We could tidy up by getting rid of it and just asserting that all the locations are the same, since I don't think we support anything else.

Reply:
If we don't need it then I'd be in favour of removing it. My real thought here was that if we know that `outloc==result_loc` has to be true so we shouldn't need both symbols.